src/main/drivers/serial_softserial.c

```diff
@@ -387,6 +394,12 @@ void processTxState(softSerial_t *softSerial)
         }
     }
 
+    if (resetBits && !softSerial->rxActive && (softSerial->port.options & SERIAL_BIDIR)) {
+        setTxSignal(softSerial, 0);
+        --resetBits;
+        return;
+    }
+
```

Comment:
Changes to softserial will have side effects. Code should be localised to VTX code.